src/MyMoneroApi.js

```diff
@@ -117,24 +103,27 @@ export class MyMoneroApi {
       view_key: privateViewKey
     })
     // asGetAddressTxs = asObject({
-    //   blockchain_height: asNumber,
-    //   scanned_block_height: asNumber,
-    //   scanned_height: asNumber,
-    //   start_height: asNumber,
-    //   total_received: asString,
-    //   transaction_height: asNumber,
-    //   transactions: asArray(asObject({
-    //     coinbase: asBoolean,
-    //     hash: asString,
-    //     height: asNumber,
-    //     id: asNumber,
-    //     mempool: asBoolean,
-    //     mixin: asNumber,
-    //     timestamp: asDate,
-    //     total_received: asString,
-    //     total_sent: asString,
-    //     unlock_time: asNumber
-    //   }))
+    //   blockchain_height: asNumber, // Current blockchain height
+    //   scanned_block_height: asNumber, // Current scan progress
+    //   scanned_height: asNumber, // Current tx scan progress
+    //   start_height: asNumber, // Start height of response
+    //   total_received: asString, // Sum of received outputs
+    //   transactions: asArray(
+    //     asObject({
+    //       coinbase: asBoolean, // True if tx is coinbase
+    //       hash: asString, // Bytes of tx hash
+    //       height: asNumber, // Block height
+    //       id: asNumber, // Index of tx in blockchain
+    //       mempool: asBoolean, // True if tx is in mempool
+    //       mixin: asNumber, // Mixin of the receive
+    //       payment_id: asString, // Bytes of tx payment id
+    //       spent_outputs: asArray(asUnknown), // List of possible spends
+    //       timestamp: asDate, // Timestamp of block
+    //       total_received: asString, // Total XMR received
+    //       total_sent: asString, // XMR possibly being spent
+    //       unlock_time: asNumber // Tx unlock time field
+    //     })
+    //   )
```

Comment:
I assume this is debug code? It'd rather put the commented code behind a flag condition to avoid rotting and linting errors from the rot. But this works for now I guess.

Reply:
This is TODO code. At some point we want to actually clean the responses, but for now we just accept whatever they send back. Maybe we should do this now.

Reply:
Maybe use the trick `if (Math.random()<0)` to block the WIP code?